RZTouchID/RZTouchID.m

```diff
@@ -96,22 +105,36 @@ - (void)addPassword:(NSString *)password withIdentifier:(NSString *)identifier c
         
         // we want the operation to fail if there is an item which needs authentication so we will use
         // kSecUseNoAuthenticationUI
-        NSDictionary *attributes = @{
-                                     (__bridge id)kSecClass: (__bridge id)kSecClassGenericPassword,
-                                     (__bridge id)kSecAttrService: [self serviceNameForIdentifier:identifier],
-                                     (__bridge id)kSecValueData: [password dataUsingEncoding:NSUTF8StringEncoding],
-                                     (__bridge id)kSecUseNoAuthenticationUI: @NO,
-                                     (__bridge id)kSecAttrAccessControl: (__bridge id)accessObject
-                                     };
+        NSDictionary *attributes = nil;
+        if ( self.touchIDMode == RZTouchIDModeBiometricKeychain ) {
+            attributes =   @{
+                             (__bridge id)kSecClass: (__bridge id)kSecClassGenericPassword,
+                             (__bridge id)kSecAttrService: [self serviceNameForIdentifier:identifier],
+                             (__bridge id)kSecValueData: [password dataUsingEncoding:NSUTF8StringEncoding],
+                             (__bridge id)kSecUseNoAuthenticationUI: @NO,
+                             (__bridge id)kSecAttrAccessControl: (__bridge id)accessObject
+                             };
+        }
+        else {
+            attributes =   @{
+                             (__bridge id)kSecClass: (__bridge id)kSecClassGenericPassword,
+                             (__bridge id)kSecAttrService: [self serviceNameForIdentifier:identifier],
+                             (__bridge id)kSecValueData: [password dataUsingEncoding:NSUTF8StringEncoding],
+                             (__bridge id)kSecUseNoAuthenticationUI: @NO
+                             };
```

Comment:
Since these two dictionaries are identical except for one param, create a mutable version with the 3 shared params and then `if ( self.touchIDMode == RZTouchIDModeBiometricKeychain )` add the 4th param